code/modules/mob/mob_defines.dm

```diff
@@ -186,3 +186,5 @@
 	var/list/additional_vision_handlers = list() //Basically a list of atoms from which additional vision data is retrieved
 
 	var/list/progressbars = null //for stacking do_after bars
+
+	var/datum/ai/ai						// Type abused. Define with path and will automagically create. Determines behaviour for clientless mobs.
```

Comment:
This is a mob-level var but is only called (and separately at that) by carbon and human `/Life()`?

Reply:
Sounds like you're proposing even more copy-paste, if you plan to continue calling AI-processing as you currently are

Reply:
I think you commented on the wrong review note, as that doesn't make sense in context here for this snippet.